resources/js/visual-editor/editor/components/editor.css

```
@@ -899,6 +957,164 @@
+.ve-block-preformatted .ve-richtext pre {
+    margin: 0;
+    padding: var(--ve-pre-padding);
+    background: var(--ve-pre-bg);
+    border-radius: var(--ve-pre-border-radius);
+    border: 1px solid var(--ve-color-border);
+    font-family: var(--ve-font-family-mono);
+    font-size: var(--ve-code-font-size);
+    line-height: 1.5;
+    color: var(--ve-pre-color);
+    white-space: pre-wrap;
+    word-wrap: break-word;
```

Stylelint (17.7.0)

[error] Expected "word-wrap" to be "overflow-wrap" (property-no-deprecated)

(property-no-deprecated)